courses/machine_learning/asl2/recommendation_systems/als_bqml.ipynb

```diff
@@ -0,0 +1,1550 @@
+{
```

Comment:
<p>Remove #standardql (it's the default) and remove the backtics (not needed unless your project name has an hyphen)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/517/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>